tally_ho/apps/tally/views/tally_manager.py

```diff
@@ -160,16 +160,20 @@ def import_rows_batch(tally,
             if line == 0 and check_file_column_names:
                 if file_to_parse_name == subconst_file_name and\
                         row != settings.SUB_CONSTITUENCY_COLUMN_NAMES:
-                    missing_columns = list(set(settings.SUB_CONSTITUENCY_COLUMN_NAMES) - set(row))
+                    missing_columns = list(
+                        set(settings.SUB_CONSTITUENCY_COLUMN_NAMES) - set(row))
                     prefix_noun = 'is' if len(missing_columns) == 1 else 'are'
                     delete_all_tally_objects(tally)
+                    message =\
+                        'Column {} {} missing in the sub constituency file'
```

Comment:
is there a way to have the placeholders marked by a name or some different kind of marker which would make it easy to identify cases where their order changes after translation.

For instance with node-i18n we could do `t("Column {{columnNames}} {{prefixNoun}} missing in the sub ...")`. 

Also does python's gettext support plurals [for instance](https://www.i18next.com/translation-function/plurals)?

Reply:
Nice catch, so Django support [pluralization](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#pluralization) translation through [ngettext()](https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.translation.ngettext) function. So we can update the logic to:
```python
ngettext('Column %(cols)s is missing in the sub constituency file',
              'Columns %(cols)s are missing in the sub constituency file',
              len(missing_columns)
) % {"cols": ', '.join(missing_columns),}
```

Reply:
Done